lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -122,14 +123,20 @@ def union(environment, a, b):
     return c
 
 
-def min(a):
+def min(a, attribute=None):
```

Comment:
Instead of re-implementing like this, please import the `min`/`max` filters from jinja2, as was outlined in https://github.com/ansible/ansible/issues/50669#issuecomment-452768986

Reply:
Just followed that PR I could and also added some unit tests the same way. Is it safe to assume that unit tests will run with the latest version of Jinja?